.envrc

```diff
@@ -28,6 +28,10 @@ export REACT_APP_OKTA_REDIRECT_URI=http://localhost:3000/implicit/callback
 # Add local paths for binaries and scripts
 PATH_add ./bin
 
+export CEDAR_ENV=impl
+export CEDAR_API_URL="https://webmethods-apigw.cedarimpl.cms.gov/gateway/Cedar%20Applications/1.1"
+export CEDAR_DIRECTORY=pkg/cedar/
+export CEDAR_SWAGGER_FILE=$CEDAR_DIRECTORY/swagger-$CEDAR_ENV.json
```

Comment:
This translates to `pkg/cedar//swagger-$CEDAR_ENV.json`, which didn't work for me